src/tests/test_national.py

```diff
@@ -248,10 +249,47 @@ def test_read_truth_national_gsp(db_session, api_client):
     db_session.add_all([gsp_yield_1_sql, gsp_yield_2_sql, gsp_yield_3_sql, gsp_sql_1])
 
     app.dependency_overrides[get_session] = lambda: db_session
+    yield db_session
 
-    response = api_client.get("/v0/solar/GB/national/pvlive/")
+    response = api_client.get("/v0/solar/GB/national/pvlive/0")
     assert response.status_code == 200
 
     r_json = response.json()
     assert len(r_json) == 3
     _ = [GSPYield(**gsp_yield) for gsp_yield in r_json]
+
+
+def test_is_fake_national_all_available_forecasts(monkeypatch, api_client):
```

Comment:
nice, these tests are in principle corretc, thank you for putting them in

Reply:
Awesome! Thanks for this. Looks like it's worked!